src/Discovery/DiscoveryDetails/DataDownloadList/DataDownloadList.tsx

```diff
@@ -22,32 +27,79 @@ const DataDownloadList = ({
   discoveryConfig,
   resourceInfo,
   sourceFieldData,
-  healLoginNeeded,
+  missingRequiredIdentityProviders,
 }: DataDownloadListProps) => {
+  const [downloadStatus, setDownloadStatus] = useState(INITIAL_DOWNLOAD_STATUS);
+  const history = useHistory();
+  const location = useLocation();
+
   const data: [] = resourceFieldValueIsValid
     ? ProcessData(sourceFieldData)
     : [];
   const noData = data.length === 0;
 
+  let userHasAccessToDownload = true;
```

Comment:
refactoring for least privilege access would make some sense, but I would defert that to a future ticket since it needs a bit for investigation.
The reason why this part is having the current behavior is to mimic what we have setup for the checkboxes in the discovery table. We are only disableing them either: 1. auth is on, and user don't have access, or 2. study has no manifest
If we are going to refactor then we are going to refactor both to ensure consistency